mdata/store_cassandra_test.go

```diff
@@ -0,0 +1,45 @@
+package mdata
+
+import (
+	"reflect"
+	"testing"
+)
+
+type testCase struct {
+	ttls         []uint32
+	windowFactor int
+	nameFormat   string
+	expected     ttlTables
+}
+
+const oneDay = 24 * 60 * 60
+const oneMonth = oneDay * 30
+const oneYear = oneMonth * 12
+
+func TestGetTTLTables(t *testing.T) {
+	tc := testCase{
+		ttls:         []uint32{oneDay, oneMonth, oneYear},
+		windowFactor: 20,
+		nameFormat:   "metric_%d",
+		expected:     make(ttlTables),
+	}
```

Comment:
so we run 1 testcase, but that 1 testcase tests a bunch of different ttl-> ttlTable mappings.
why not just test each mapping individually?
you could also make this more concise and simultaneously more extensive. something like : 

```
type testCase struct {
ttl uint32
windowFactor int
table string
window uint32
}
cases := []testCase{
{24 * 60 *60, 20, metric_16, 1},
{30 * 24 * 60 *60, 20, metric_512, 26},
...
}
```